<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="mainLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="pathWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>90</height>
       </size>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>761</width>
         <height>91</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="pathLayout" columnstretch="0,0,0,0,0,0,0,0,0,0,0">
        <item row="1" column="10">
         <widget class="QPushButton" name="btn_findLocalMusic">
          <property name="text">
           <string>匹配音乐</string>
          </property>
         </widget>
        </item>
        <item row="0" column="10">
         <widget class="QPushButton" name="btn_findMusic">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>1</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>45</height>
           </size>
          </property>
          <property name="text">
           <string>获取歌单</string>
          </property>
         </widget>
        </item>
        <item row="2" column="10">
         <widget class="QPushButton" name="btn_Lrc">
          <property name="text">
           <string>生成歌词</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>播放器目录</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QLineEdit" name="txt_musicDir">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="btn_chooseMusicPath">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#btn_chooseMusicPath
{
border-image:(&quot;./images/open.png&quot;)
}</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>本地音频目录</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="9">
         <widget class="QPushButton" name="btn_cookie">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>cookie</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>歌单链接</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="8">
         <widget class="QLabel" name="ListInfo">
          <property name="text">
           <string>暂时稳定版v1.024，如有BUG，纯属故意</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1" colspan="9">
         <widget class="QLineEdit" name="ListID">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="9">
         <widget class="QPushButton" name="btn_choosePlayerPath">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="2" column="8">
         <widget class="QPushButton" name="btn_Edit">
          <property name="text">
           <string>编辑列表</string>
          </property>
         </widget>
        </item>
        <item row="1" column="6" colspan="3">
         <widget class="QLineEdit" name="txt_playerDir">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QWidget" name="pushWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>761</width>
         <height>42</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="pushLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>微博：https://weibo.com/shihoongbo</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_state">
          <property name="text">
           <string>欢迎使用</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_copyMusic">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>导入至WALKMAN</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>序号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>歌曲文件名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>歌曲本地地址</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>歌曲匹配</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>歌词匹配</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>导入状态</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
